Shared/Samples/Authenticate with Integrated Windows Authentication/README.md

```diff
@@ -0,0 +1,48 @@
+# Authenticate with Integrated Windows Authentication
```

Comment:
Add "Authenticate with Integrated Windows Authentication" to this list: https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/beab5d441fac2da56d2cad29fd849226aa0f14be/Scripts/CI/common.py#L25 so the title is in the exception.